addons/l10n_be_invoice_bba/data/mail_template_data.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="account.email_template_edi_invoice" model="mail.template">
+            <field name="body_html"><![CDATA[
+% set access_action = object.get_access_action()
+% set access_url = access_action['type'] == 'ir.actions.act_url' and access_action['url'] or '/report/pdf/account.report_invoice/' + str(object.id)
+
+<p>Dear ${object.partner_id.name}
+% if object.partner_id.parent_id:
+    (<i>${object.partner_id.parent_id.name}</i>)
+% endif
+,</p>
+<p>Here is your invoice <strong>${object.number}</strong>
+% if object.origin:
+(with reference: ${object.origin} )
+% endif
+amounting in <strong>${object.amount_total} ${object.currency_id.name}</strong>
+from ${object.company_id.name}.
+</p>
+
+    <br/><br/>
+    <center>
+      <a href="${access_url}" style="background-color: #1abc9c; padding: 20px; text-decoration: none; color: #fff; border-radius: 5px; font-size: 16px;" class="o_default_snippet_text">View Invoice</a>
+    </center>
+    <br/><br/>
+
+% if object.reference_type=='bba' and object.reference:
+    <p>Please use the following communication for your payment : <b>${object.reference}</b></p>
+% endif
+
+% if object.state=='paid':
+    <p>This invoice is already paid.</p>
```

Comment:
Remove this part. It should not be linked to the status (keep it simple), it's rare to re-send the invoice if it has been already paid. If a customer wants a copy, it is interesting to have the communication because it helps tracking the payment.
Moreover, I don't think it's working.